<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1912</width>
    <height>1051</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currentWord">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>46</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>790</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>940</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtRight">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Правильно</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLoose">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>630</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ошибок</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cntRight">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>720</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cntLoose">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>710</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>960</y>
      <width>921</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="txtProgress">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>900</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прогресс</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputLetter">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>360</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QFrame" name="winFrame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>991</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="txtItog">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>39</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Итог:</string>
     </property>
    </widget>
    <widget class="QListView" name="FalsListView">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>751</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="txtCnt">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <zorder>inputLetter</zorder>
   <zorder>currentWord</zorder>
   <zorder>startButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>txtRight</zorder>
   <zorder>txtLoose</zorder>
   <zorder>cntRight</zorder>
   <zorder>cntLoose</zorder>
   <zorder>progressBar</zorder>
   <zorder>txtProgress</zorder>
   <zorder>status</zorder>
   <zorder>winFrame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
